### PR TITLE
Update to Xenial on Travis

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

**Comment by dhalbert** [comment]
well, not faster :frowning_face: 

trusty:
image

xenial:

image

File: .travis.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ before_script:
 
   # report some good version numbers to the build
   - gcc --version
-  - (! var_search "${TRAVIS_SDK-}" elf || arm-none-eabi-gcc --version)
+  - (! var_search "${TRAVIS_SDK-}" arm || arm-none-eabi-gcc --version)
```
Comment (jepler) [inline .travis.yml]: Ooops! thanks for spotting this.  Could be its own PR, if the update to enial is held back for any other reason.
